media-sound/mixxx/mixxx-2.5.3.recipe

```diff
@@ -0,0 +1,179 @@
+	lib:libQt6Gui$secondaryArchSuffix
+	lib:libQt6Keychain$secondaryArchSuffix
+	lib:libQt6Network$secondaryArchSuffix
+	lib:libQt6OpenGL$secondaryArchSuffix
+	lib:libQt6Qml$secondaryArchSuffix
+	lib:libQt6Quick$secondaryArchSuffix
+	lib:libQt6ShaderTools$secondaryArchSuffix
+	lib:libQt6Sql$secondaryArchSuffix
+	lib:libQt6Svg$secondaryArchSuffix
+	lib:libQt6Test$secondaryArchSuffix
+	lib:libQt6Widgets$secondaryArchSuffix
+	lib:libQt6Xml$secondaryArchSuffix
+	lib:librubberband$secondaryArchSuffix
+	lib:libsndfile$secondaryArchSuffix
+	lib:libSoundTouch$secondaryArchSuffix
+	lib:libsqlite3$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libtag$secondaryArchSuffix
+	lib:libusb_1.0$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libvorbisfile$secondaryArchSuffix
+	lib:libvorbisenc$secondaryArchSuffix
+	lib:libwavpack$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	ffmpeg6${secondaryArchSuffix}_devel
+	devel:ms_gsl
+#	devel:libbenchmark$secondaryArchSuffix
+	devel:libchromaprint$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libebur128$secondaryArchSuffix
+#	devel:libboost_$secondaryArchSuffix
+	devel:libfftw3$secondaryArchSuffix
+	devel:libFLAC$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libgtest$secondaryArchSuffix
+	devel:libhidapi$secondaryArchSuffix
+	devel:libid3tag$secondaryArchSuffix
+	devel:liblilv_0$secondaryArchSuffix
+	devel:libmad$secondaryArchSuffix
+	devel:libmodplug$secondaryArchSuffix
+	devel:libmp3lame$secondaryArchSuffix
+	devel:libogg$secondaryArchSuffix
+	devel:libopusfile$secondaryArchSuffix
+	devel:libportaudio$secondaryArchSuffix
+	devel:libportmidi$secondaryArchSuffix
+	devel:libprotobuf$secondaryArchSuffix >= 31
+	devel:libQt6Core$secondaryArchSuffix
+	devel:libQt6Core5Compat$secondaryArchSuffix
+	devel:libQt6Keychain$secondaryArchSuffix
+	devel:libQt6Qml$secondaryArchSuffix
+#	devel:libQt6Quick$secondaryArchSuffix
+#	devel:libQt6ShaderTools$secondaryArchSuffix
+	devel:librubberband$secondaryArchSuffix >= 3
+	devel:libsndfile$secondaryArchSuffix
+	devel:libsoundtouch$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libtag$secondaryArchSuffix >= 2.0
+	devel:libusb_1.0$secondaryArchSuffix
+	devel:libvorbis$secondaryArchSuffix
+	devel:libwavpack$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:ccache
+	cmd:cmake
+	cmd:doxygen
+	cmd:gcc$secondaryArchSuffix
+	cmd:lld >= 20
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+BUILD()
+{
+#	rm -rf build
+	mkdir -p build/downloads src/arch
+	cp $sourceDir2/libdjinterop-0.24.3.tar.gz build/downloads
+	cp $sourceDir3/libkeyfinder-2.2.8.zip build/downloads
+	# workaround for: fatal error: arch/1/limits.h: No such file or directory
+	# ticket created at: https://dev.haiku-os.org/ticket/19765
+	ln -s /boot/system/develop/headers/posix/arch/x86_64/ src/arch/1
```

Comment:
Untested, but I guess this would be better (if it works):

```suggestion
	ln -s $(finddir B_SYSTEM_HEADERS_DIRECTORY)/posix/arch/$effectiveTargetArchitecture/ src/arch/1
```

Someone should maybe test this on x86 and riscv64.